src/SplitPackages/CsprojFileBuilder.cs

```diff
@@ -114,39 +113,79 @@ public void AddImports(string fx, string import)
 
         public void Execute()
         {
-            var document = new JObject();
-            document["dependencies"] = JObject.FromObject(CreateDependenciesDictionary(_dependencies));
-            document["frameworks"] = JObject.FromObject(CreateFrameworksDictionary());
-            document["runtimes"] = JObject.FromObject(CreateRuntimesDictionary());
-
-            var writer = _whatIf ? StreamWriter.Null : File.CreateText(_path);
-            using (var jsonWriter = new JsonTextWriter(writer) { Formatting = Formatting.Indented })
+            var document = new XDocument();
+            var dependenciesDictionary = CreateDependenciesDictionary(_dependencies);
+            var packageDependencies = CreatePackageReferenceList(dependenciesDictionary);
+            var root = new XElement("Project", new XAttribute("Sdk", "Microsoft.NET.Sdk"), new XAttribute("ToolsVersion", "15.0"));
+            var propertyGroup = new XElement("PropertyGroup");
+            propertyGroup.Add(new XElement("TargetFrameworks", string.Join(";", _frameworks.Select(i => Frameworks.GetMoniker(i.Name)))));
```

Comment:
Give this variable a meaningful name like projectDefinitionPropertyGroup or simply projectDefinition but something that indicates what you are trying to achieve with this. The problem is property group is very generic and makes it hard to know what its being used for.